Modelica/Blocks/Logical.mo

```diff
@@ -38,20 +52,50 @@ the output is <strong>false</strong>.
 </html>"));
   end Or;
 
-  block Xor "Logical 'xor': y = u1 xor u2"
+  block MultiOr "Logical or of Boolean vector elements"
+    extends Modelica.Blocks.Interfaces.PartialBooleanMISO;
+  equation
+    y=Modelica.Math.BooleanVectors.anyTrue(u);
+    annotation (Icon(coordinateSystem(initialScale=0.1),
+      graphics={ Text(extent={{-90,40},{90,-40}},textString="or")}),
+      Documentation(info="<html>
+<p>
+The output is <strong>true</strong> if any of the elements of the input vector are <strong>true</strong>, otherwise
+the output is <strong>false</strong>.
+</p>
+</html>"));
+  end MultiOr;
+
+  block Nand "Logical 'nand': y = not (u1 and u2)"
     extends Blocks.Interfaces.partialBooleanSI2SO;
   equation
-    y = not ((u1 and u2) or (not u1 and not u2));
+    y = not (u1 and u2);
     annotation (Icon(coordinateSystem(preserveAspectRatio=true, extent={{-100,-100},
               {100,100}}), graphics={Text(
             extent={{-90,40},{90,-40}},
-            textString="xor")}), Documentation(info="<html>
+            textString="nand")}), Documentation(info="<html>
 <p>
-The output is <strong>true</strong> if exactly one input is <strong>true</strong>, otherwise
+The output is <strong>true</strong> if at least one input is <strong>false</strong>, otherwise
```

Comment:
This could be formulated more consistently with `MultiOr` (since there is always a non-zero number of inputs):
```suggestion
The output is <strong>true</strong> if any of the inputs is <strong>false</strong>, otherwise
```

Reply:
I don't see that there has to be at non-zero number of inputs.

Reply:
We have `u1` and `u2`, which adds up to two inputs, no more, no less.